### PR TITLE
created game grid

### PR DESCRIPTION
**Pull request by Matthew-Priest**

### CODE REVIEW

File: src/gamegrid.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+let rows = 20;
+let cols = 10;
+let gamegrid = new Array(rows).fill(0).map(() => new Array(cols).fill(0));
+for (let y = 0; y < 20; y++) {
+    for (let x = 0; x < 10; x++) {
```
Comment (samyou-softwire) [inline src/gamegrid.js:5]: Perhaps we should use our consts in all cases
```suggestion
for (let y = 0; y < rows; y++) {
    for (let x = 0; x < cols; x++) {       
```

File: src/gamegrid.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+let rows = 20;
+let cols = 10;
+let gamegrid = new Array(rows).fill(0).map(() => new Array(cols).fill(0));
```
Comment (samyou-softwire) [inline src/gamegrid.js]: As these aren't reassigned it'd be better to mark these as const.

Note that you can edit an item in an array while it's const, you just can't reassign a whole new array in its place
```suggestion
const rows = 20;
const cols = 10;
const gamegrid = new Array(rows).fill(0).map(() => new Array(cols).fill(0));
```

**Review by samyou-softwire — COMMENTED** [review]
Good work :) Nice use of a map to fill the array. As mentioned on #3, make sure to stay consistent on the casing of variable names, ie `gameGrid`.

File: src/gamegrid.js (hunk 1 above)
Comment (Matthew-Priest) [reply src/gamegrid.js:5]: we have removed this piece of code and replaced it.

File: src/gamegrid.js (hunk 2 above)
Comment (Matthew-Priest) [reply src/gamegrid.js]: change has been made. thanks

**Pull request closed without merging**